src/main/java/com/trustev/domain/entities/Case.java

```diff
@@ -52,6 +52,10 @@ public String getCaseNumber() {
 		return caseNumber;
 	}
 	
```

Comment:
Ideally we would not allow this. The idea was that a case always should have a CaseNumber so they should set it via the constructor